kube-config-files/public-things-api-service.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name:  public-things-api
+  labels:
+    name:  public-things-api
```

Comment:
please use "app":  public-things-api label here, since we select deployments based on this label